### ISSUE TITLE
Vehicle ID

### ISSUE THREAD

**Issue by LouDnl** [micovery/ArmA3_Wasteland.Altis#74]
On extDB when using info the vehicle ID is not showing/not working.

**Comment by micovery** [comment]
Vehicle IDs are only assigned on saved/restored vehicles, so you won't see an ID on a vehicle has not not been saved.

There is some code already that fetches the vehicle ID for display. See: https://github.com/micovery/ArmA3_Wasteland.Altis/blob/910b13dd4a5d7e1e3992bdcbb6a36ea130d7d455/addons/vactions/va_functions.sqf#L171.

The only issue I could think is that the data type for the ID in extDB is not a string. Can you verify what the data type for vehicle IDs is in extDB?

**Comment by LouDnl** [comment]
How would I find that out? using cursortarget getVariable "A3W_vehicleID"; on a vehicle returns the ID number and nothing more. No "" or anything.

**Comment by micovery** [comment]
Sounds like it's a number then. You can try doing: 

``` SQF
player groupChat format["%1", typeName (cursortarget getVariable "A3W_vehicleID")];
```

**Comment by LouDnl** [comment]
It says SCALAR.

**Comment by micovery** [comment]
Fixed by 3b371e3268d4a8f3f11c414ac0e93c3716d1f72d
